source/_drafts/Video-js-7-4.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Video.js 7.4
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+  - a11y
+  - responsive
+  - languages
+  - live-ui
+  - focus-visible
+---
+
+It's time to have an overview of Video.js 7.4, first released early December.
+The big new feature for this release is a UI that allows you to seek during live streams.
```

Comment:
I would make this sentence and the one below bullet points to read a bit easier

Reply:
we have a more details bullet point list below, was going to keep this as text.